From e015f21d9f3a86f3b6b71f462ab637a6fc126a89 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio.traversaro@iit.it>
Date: Tue, 19 Apr 2022 19:19:03 +0200
Subject: [PATCH] Do not search for Development component when searching for
 Python3

---
 python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 1c21d8c..dfabd46 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -4,7 +4,7 @@
 
 if(ICUB_MODELS_USES_PYTHON)
 
-  find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+  find_package(Python3 COMPONENTS Interpreter REQUIRED)
 
   option(ICUB_MODELS_DETECT_ACTIVE_PYTHON_SITEPACKAGES
     "Do you want icub-models to detect and use the active site-package directory? (it could be a system dir)"
